tests/ccl_test_pyccl_interface.py

```diff
@@ -925,6 +925,13 @@ def test_halomod():
     for cosmo_mg in reference_models_mg():
         yield check_massfunc_MG, cosmo_mg
 
+def test_haloprofile():
```

Comment:
Why does this appear in this PR?

Reply:
For me it's not showing up as a change in this PR. It's just lines 925 - 927 above it. Do you see line 928 as a new line in this PR?

Reply:
Oh, it must have been brought in with master. I'm just viewing my changes since I last reviewed, so those show up as well.